pkg/ggql/field.go

```diff
@@ -90,7 +90,9 @@ func (f *Field) write(buf *bytes.Buffer, depth int) {
 			if 0 < i {
 				_, _ = buf.WriteString(", ")
 			}
-			_ = av.Write(buf)
+			if av != nil {
+				_ = av.Write(buf)
```

Comment:
If you're writing  a representation here and `av` is nil, you would have a blank space in between the commas. Do you want to represent the nil value instead? Or maybe exclude it from the list

Reply:
fixed